web3/_utils/method_formatters.py

```diff
@@ -0,0 +1,471 @@
+import codecs
+import operator
+
+from eth_utils.curried import (
+    apply_formatters_to_sequence,
+    is_address,
+    is_bytes,
+    is_dict,
+    is_integer,
+    is_null,
+    is_string,
+    remove_0x_prefix,
+    text_if_str,
+    to_checksum_address,
+    to_tuple,
+)
+from eth_utils.toolz import (
+    complement,
+    compose,
+    curried,
+    curry,
+    partial,
+)
+from hexbytes import (
+    HexBytes,
+)
+
+from web3._utils.abi import (
+    is_length,
+)
+from web3._utils.encoding import (
+    hexstr_if_str,
```

Comment:
this is available from `eth_utils` (this would be a good "Good First Issue" ticket to open)